### ISSUE TITLE
Watch for Docker events on image pull to display deploy progress in UI

### ISSUE THREAD

**Issue by andrey18106** [cloud-py-api/app_api#185]
### Describe why it is important and where it will be useful

ExApps packaged in Docker container (especially with AI features) could have pretty big image sizes, as well as initialization data to download.
It's important to keep the ExApp "Deploy and enable" progress visible to the user.

This is also crucial in case of a slow download speed due to both slow user connection speed or slow container registry.

### Describe your proposed solution

Implement watch and filtering out the [docker events](https://docs.docker.com/engine/api/v1.41/#tag/System/operation/SystemEvents) (pull events) to keep track and show the progress.

### Additional context

For UI this will also allow to restore the download progress state after page refresh.

**Comment by bigcat88** [comment]
We will continuously receive reports for failing installations(like #219) until we implement this.

On my ISP with 300Mb it takes for 5 minutes to download ExApp Image(6 gigabytes) with CUDA libraries.
And during all this 5 minutes we have no information what is happening.

We can't create a record in `oc_ex_apps` table to store some info, as that table contains only ExApps related info.

I propose to implement additional logic for ExApps deploy:

1. New table `oc_ex_apps_deploying` with `id, appid, created_time, last_update_time, finished, error, status` where `appid` and `error` are strings, `status` is json and all other fields are integers. 
`finished` is flag with values 0 or 1.
2. We will made `pullContainer` function run in background with `occ`, it starts pulling and update information in table by appid.
3. We implement simple routes for **frontend** to  get information from this table, to not allow two parallel clicking on a button: "Deploy and enable" and display overall progress or error in realtime.

Format of the `status`:
```
{
   {layer_id: string => progress: int},
   {layer_id: string => progress: int}
}
```
e.g:
```
{
    {"30d855997954": 0},
    {"75e2b45cbee5": 22},
    {"4f4fb700ef54": 44},
    {"1c5dae15d6ea": 3},
}
```
This will allow you to calculate the approximate overall progress in % in the UI.


This should not be too hard to implement, or do I missing something?

**Comment by bigcat88** [comment]
Implemented in #233
